services/brig/src/Brig/API/User.hs

```diff
@@ -315,6 +318,19 @@ createUser new@NewUser{..} = do
             forM_ ucs $ Intra.onConnectionEvent uid Nothing . toEvent
             Data.deleteInvitation (inInviter inv) (inInvitation inv)
 
+    addUserToTeamSSO :: UserAccount -> TeamId -> UserIdentity -> ExceptT CreateUserError AppIO ()
+    addUserToTeamSSO account tid ident = do
+        let uid = userId (accountUser account)
+        added <- lift $ Intra.addTeamMember uid tid
+        unless added $
+            throwE TooManyTeamMembers
+        lift $ do
+            activateUser uid ident
+            void $ onActivated (AccountActivated account)
+            Log.info $ field "user" (toByteString uid)
+                     . field "team_id" (toByteString tid)
```

Comment:
Please use field `team` here

Reply:
done, but only for logging.  json (`team_id`) and cql (`already `team`) unchanged.  is that what you wanted?